README.md

```diff
@@ -1 +1,10 @@
-# santasonic
\ No newline at end of file
+# santasonic
+
+TODOS:
+- wymyślić strukture repo. moja propozycja jest taka, żeby mieć folder projects,
+  w projects mieć już podfolderki, które najlepiej jakby były nazywane sprytnie. Np. prosty program, który blinkuje diodą żeby nazywał się np. simple_blink.
+  W środku każdego projekciku fajnie jakby znajdowały się dokładnie trzy rzeczy:
+  1. kod programu nazwany dowolnie np. simple_blink.py
+  2. folder z wymaganymi bibliotekami jeśli takie są
```

Comment:
Z tymi bibliotekami to mówię, bo ja ostatnio miałem problem żeby przy uzyciu thonnyego zainstalować bibliotekę i musiałem ręcznie po prostu plik przekopiować na pico i zadziałało.